spec/spec_helper.rb

```diff
@@ -13,13 +13,16 @@
 require "rspec/collection_matchers"
 require "fakeredis/rspec"
 require "database_cleaner"
-require 'simplecov'
+require "simplecov"
 SimpleCov.start
 
 ENV["MONGOID_ENV"] = "test"
 Mongoid.load!("#{File.dirname(__FILE__)}/config/mongoid.yml")
 
-Combustion.initialize! :active_record
+begin
+  Combustion.initialize! :active_record
```

Comment:
How come we need to wrap this with an empty rescue block?  Shouldn't this always work since we're requiring combustion?